app/timetables/models.py

```diff
@@ -1,3 +1,10 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from app.shared.models import CommonInfo
```

Comment:
Please give a line spacing before the app import, and two line spacing after all imports before the class definition.